crates/sui-e2e-tests/tests/multisig_tests.rs

```diff
@@ -175,8 +178,15 @@ async fn test_multisig_e2e() {
 
 #[sim_test]
 async fn test_multisig_with_zklogin_scenerios() {
-    let test_cluster = TestClusterBuilder::new().with_default_jwks().build().await;
+    let test_cluster = TestClusterBuilder::new()
+        .with_default_jwks()
+        .with_epoch_duration_ms(1000)
+        .build()
+        .await;
     test_cluster.wait_for_authenticator_state_update().await;
+    test_cluster
+        .wait_for_epoch_with_timeout(Some(9), Duration::from_secs(190))
```

Comment:
why do we need to wait 9 epochs in this test?

Reply:
because the zk proof is generated for max epoch = 10.